master/internal/command/command.go

```diff
@@ -265,8 +265,6 @@ func (c *command) Receive(ctx *actor.Context) error {
 			}
 		}
 
-		c.eventStream, _ = ctx.ActorOf("events", newEventManager(c.Config.Description))
-
 		var eventStreamConfig *sproto.EventStreamConfig
 		if c.eventStream != nil {
```

Comment:
hmm I prefer to leave more go changes out of here (aka keep it as is) if it can be left out cleanly so lets remove the rest of eventStream references in your pr if that sounds good to you.